### PR TITLE
Added ETHAnglia Community in Norwich, UK

### PR DESCRIPTION
**Pull request by JAdams-eth**

### CODE REVIEW

File: src/data/community-meetups.json [hunk 1]
```diff
@@ -299,4 +299,10 @@
     "location": "Dubai",
     "link": "https://www.meetup.com/defi-dubai/"
   }
```
Comment (corwintines) [inline src/data/community-meetups.json:301]:
```suggestion
  },
```

**Review by minimalsm — APPROVED** [review]
This is building fine locally for me. No idea why Gatsby is freaking out here but I'm confident there is no issue, so merging.

Thanks for the PR @JAdams-eth 

Be sure to [claim your POAP](https://ethereum.org/en/contributing/#poap) also 🥳!

**Comment by minimalsm** [comment]
@all-contributors please add @JAdams-eth for docs

**Comment by minimalsm** [comment]
@all-contributors please add @JAdams-eth for docs

**Pull request merged**